src/Take.Blip.Builder/Actions/ExecuteScript/ExecuteScriptAction.cs

```diff
@@ -44,7 +44,7 @@ public override async Task ExecuteAsync(IContext context, ExecuteScriptSettings
             }
             catch (Exception e)
             {
-                _logger.Information(e.Message);
+                _logger.Information("{Message}", e.Message);
```

Comment:
Maybe this error can be more usefull if we use a fixed message and pass the exception as the first parameter?
```suggestion
                _logger.Information(e, "Error converting timezone");
```

Reply:
Fixed, thanks